Show 100% done if there are no tasks

Senatrius opened the pull request:
PR to close #1374

### **Changes**
- Add a ternary operator to show done percentage if there are any tasks that day and show 100 percent if there aren't

### **Comment**
I am wondering if instead of showing "100% done" if there are no tasks added that day it would be better to show "No tasks today" instead, or something like that.

tijlleenders commented:
@Senatrius 
> I am wondering if instead of showing "100% done" if there are no tasks added that day it would be better to show "No tasks today" instead, or something like that.  

That would indeed be better. Good suggestion!

Tushar-4781 commented:
hi @Senatrius, glad to see that you resolved that bug in addition to this can you add some padding below to this progress bar in case there are no tasks.

![image](https://github.com/tijlleenders/ZinZen/assets/39093015/9a2df830-91a9-42ed-9a2d-83b4cd5b8229)

tijlleenders reviewed (approved):
Thanks a bunch ;)

Senatrius commented:
Replaced the message, now it shows "no tasks today" if no tasks were assigned that day. As for the padding, I've added it to the task list. 

My reasoning is that adding the padding to the list makes sure that it only applies when the task list is expanded and it doesn't add any extra height to the div when the list is collapsed.

tijlleenders commented:
Totally agree, nice one!
